### ISSUE TITLE
What is the correct way to use Adafactor?

### ISSUE THREAD

**Issue by alexvaca0** [huggingface/transformers#9961]
Hi, from the papers I've seen that Adafactor is typically used with no learning rate (as in Pegasus paper), however, when I try to execute run_seq2seq.py or seq2seq/finetune_trainer.py from your examples, and set --adafactor parameter, without specifying learning rate (for no learning rate), it uses the default 3e-05. Is there a way to use Adafactor without learning rate?
